mypy/util.py

```diff
@@ -134,3 +140,36 @@ def id(self, o: object) -> int:
             self.id_map[o] = self.next_id
             self.next_id += 1
         return self.id_map[o]
+
+
+if sys.version_info >= (3, 6):
+    PathType = Union[AnyStr, os.PathLike]
```

Comment:
Hmm, actually it's not going to work in runtime because `PathLike` derives only from `abc.ABC`, and there's no equivalent thing exported from `typing`. I'll use AnyStr for now.